pycsw/wsgi.py

```diff
@@ -205,6 +207,12 @@ def get_configuration_path(process_environment, request_environment,
         )
     if not os.path.isabs(configuration_path):
         configuration_path = os.path.join(pycsw_root, configuration_path)
+
+    # brute force try default.cfg in the installation directory
+    default_cfg = os.path.join(pycsw_root, 'default.cfg')
+    if os.path.isfile(default_cfg):
```

Comment:
@ricardogsilva even in the 4-minute install case `configuration_path` indeed exists.  The above checks if `default_cfg` exists, which is an assumption that the setup has consciously created this file on disk.  Other options?